src/Features/Core/Portable/PullMemberUp/MemberAndDestinationValidator.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.PullMemberUp
+{
+    internal static class MemberAndDestinationValidator
+    {
+        internal static bool IsDestinationValid(Solution solution, INamedTypeSymbol destination, CancellationToken cancellationToken)
+        {
+            if (destination == null)
+            {
+                return false;
```

Comment:
This case actually is impossible to happen (ideally). I add this check since I am started to use destination symbol later. So should we throw some exceoption here ?